reana_client/openapi_connections/reana_server.json

```diff
@@ -346,13 +346,159 @@
+            "in": "formData",
+            "name": "file_content",
+            "required": true,
+            "type": "file"
+          },
+          {
+            "description": "Required. File name.",
+            "in": "query",
+            "name": "file_name",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Request succeeded. File successfully trasferred.",
+            "examples": {
+              "application/json": {
+                "message": "File successfully transferred"
+              }
+            },
+            "schema": {
+              "properties": {
+                "message": {
+                  "type": "string"
+                }
+              },
+              "type": "object"
+            }
+          },
+          "400": {
+            "description": "Request failed. The incoming data specification seems malformed"
+          }
+        },
+        "summary": "Seeds the analysis workspace with the provided file."
+      }
+    },
+    "/api/analyses/{analysis_id}/workspace/code/": {
+      "get": {
+        "description": "This resource is expecting an analysis UUID to return its list of code files.",
+        "operationId": "get_analysis_code",
+        "parameters": [
+          {
+            "description": "Required. Organization which the analysis belongs to.",
+            "in": "query",
+            "name": "organization",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "Required. UUID of analysis owner.",
+            "in": "query",
+            "name": "user",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "Required. analysis UUID.",
+            "in": "path",
+            "name": "analysis_id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Requests succeeded. The list of code files has been retrieved.",
+            "schema": {
+              "items": {
+                "properties": {
+                  "last-modified": {
+                    "format": "date-time",
+                    "type": "string"
+                  },
+                  "name": {
+                    "type": "string"
+                  },
+                  "size": {
+                    "type": "integer"
+                  }
+                },
+                "type": "object"
+              },
+              "type": "array"
+            }
+          },
+          "400": {
+            "description": "Request failed. The incoming data specification seems malformed."
```

Comment:
"incoming data" can sound perhaps misleading to people in the context of "code" endpoints? 

We have three possibilities: (1) we can keep it like this hoping it isn't too confusing; (2) we can amend the verbiage to be more context-specific for each endpoint (using terms inputs/code/outputs); (3) we can replace the term with some neutral speak ("incoming payload") that will be applicable to all 400 errors regardless of the endpoint.